plugins/utils/redirects.js

```diff
@@ -177,6 +177,8 @@ const redirects = `
   /docs/current/developer-docs/production/instruction-limits /docs/current/developer-docs/production/resource-limits
   /docs/current/developer-docs/backend/resource-limits /docs/current/developer-docs/production/resource-limits
   /docs/current/concepts/data-centers /docs/current/concepts/
+  /docs/current/developer-docs/build/languages/rust/rust-intro/ /docs/current/developer-docs/backend/rust/
```

Comment:
I think we want to omit the trailing slash, otherwise it will error out.

Reply:
(sorry for giving you an imprecise mapping earlier)

Reply:
No worries, thanks for catching it! 😃